templates/includes/_footer.html.twig

```diff
@@ -0,0 +1,40 @@
+<footer class="bg-secondary text-center text-primary">
+    <div class="container p-4">
+        <div class="row justify-content-center">
+            <div class="offset-3"></div>
+            <div class="col-lg-3 col-md-6 mb-4 mb-md-0 d-flex flex-column justify-content-center">
+            <h6 class="text-uppercase fw-bold mb-4">Jobpermut</h6>
+            </div>
+            <div class="col-lg-3 col-md-6 mb-4 mb-md-0">
+                <div class="mb-0 d-flex justify-content-center">
+                    <div class="mb-0 d-flex flex-column justify-content-start align-items-start text-decoration-none">
+                        <h5 class="mb-4">Liens</h5>
+                        <div>
+                            <a class="text-decoration-none" href="{{ path('home') }}" class="text-white">Accueil</a>
+                        </div>
+                        <div>
+                            <a class="text-decoration-none" href="#!" class="text-white">A propos</a>
+                        </div>
+                        <div>
+                            <a class="text-decoration-none" href="#!" class="text-white">Connection</a>
+                        </div>
+                        <div>
+                            <a class="text-decoration-none" href="#!" class="text-white">Inscription</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="col-lg-3 col-md-6 mb-4 mb-md-0 ">
+                <div class="mb-0 d-flex flex-column align-items-start">
+                    <h5 class="mb-4">Contactez nous</h5>
+                    <p class="text-start">Bureau 562 78 Avenue des Champs-Élysées, 75008 Paris</p>
+                    <p class="text-start">02 38 48 39 48</p>
+                    <p class="text-start">contact@jobpermut.fr</p>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <p class="text-center p-3">Copyright © 2021 : Jobpermut. Tous droits réservés.</p>
```

Comment:
oh really?are you sure about the legal aspect of this? because I am not :upside_down_face:

Reply:
yes we are, at this point the customer hasn't registered his trademark but it is in progress, since we working on it, it's worth for us to display it now